docs/reference/commandline/version.md

```diff
@@ -18,38 +18,97 @@ Options:
 
 ## Description
 
-By default, this will render all version information in an easy to read
-layout. If a format is specified, the given template will be executed instead.
+The `version` command prints detailed information about the version of the
+`docker` CLI, the Docker Engine (`dockerd`), as well as other components.
 
-Go's [text/template](https://golang.org/pkg/text/template/) package
-describes all the details of the format.
+By default, this renders all version information in an easy to read
+layout. Use the [`--format`](#format) option to customize the output with a Go
+template.
 
-## Examples
+> **Note**
```

Comment:
Note that the version command [`Docker version`] outputs version numbers for Docker components, while the --version flag [`Docker --version`] output the version number for Docker Engine CLI you are using. Can we say Docker Engine CLI instead of Docker CLI or is it innacurate?